apps/frontend/app/(with-header)/profile/page.tsx

```diff
@@ -3,7 +3,7 @@ import { requireAuthorizedUser } from 'src/utils/auth.util';
 
 export default async function ProfilePage() {
   // TODO: use NextAuth App Router guide for caching
-  await requireAuthorizedUser();
+  await requireAuthorizedUser({ currentRoute: '/profile' });
```

Comment:
Please try the following solution to avoid passing route directly
https://github.com/vercel/next.js/issues/46618#issuecomment-1450416633